settings.json

```diff
@@ -849,6 +849,16 @@
       }
     }
   ],
+  "nix": [
+    {
+      "command": "rnix-lsp",
+      "url": "https://github.com/nix-community/rnix-lsp",
+      "description": "Language server implementation for Nix",
+      "requires": [
+        "nix"
```

Comment:
nix-env is also required?

Reply:
Yes, but as far as I know nix-env command is included into nix, so having nix means that you also have nix-env